_posts/2019-12-19-sw-ix-review.md

```diff
@@ -0,0 +1,84 @@
+
+A few egregious examples:
+
+1. Giving life
+
+As I said before the whole Anakyn story arc has been destroyed. Now anyone can heal. Force heal is not a new thing, but before it was something really hard to do. Now even bringing people back to life can be done and the whole Darth Plaiguis the Wise thing has disappeared. In the old republic with Jedi in peace and studying for millenia they were never able to crack it, not Windu, not Yoda, but we are to believe than now magically it can be done so easily.
+
+2. Dementors
+
+Now Palpatine has become a dementor. Stealing life force is acceptable and if this was the only thing happening in this movie I would not have complained, it is when it's put together with all the rest that makes you think wtf.
+
+3. Force lightning
+
+Force lightning can now destroy a whole fleet. That same force lightning cannot kill one single Jedi. And can beoverpowered by 2 lightsabers, neat!
+
+## Palpatine
+
+The whole Palpatine thing was ridiculous. He disappeared for 30 years and now he comes back out of knowhere, no forthshadowing at all. He gets introduces in the scrolling credits... And when he does he lasts less than three hours and is dead by the end of the movie.
+
+Palpatine has always been a real strategist, especially during the Original Trilogy as we've seen him rise to power. His behaviour here is totally not in line with the character. After having his strenght restored and neutralizing a whole fleet by himself he gets beaten by 2 lightsabers reflecting his lightning back to him in a flashback of Episode 3... We didn't even get a decent fight scene.
+
+## Copy of Ep 6
+
+This is not as egregious as The Force Awakens in which almost the whole movie was a copy paste from episode 4, but the final scene of the vault opening to reveal the rebel fleet losing and Ray having to take the throne to save her friends was so cringy. Try to make it look like you're at least trying...
+
+## Others
+
+- Lando doesn't do anything and doesn't add anything to the movie, was nice to see him back though. They had him go around the galaxy asking for help with Chewy...[^1]
+
+- Fin needs to say something to Rae and that whole plot is abandoned.
+
+- The knights Kylo of appear out of knowhere and go nowhere. They don't even have energy sabers, what they have looks like can be cut off by any lightsaber with ease.
+
+- In the end Ray goes to Tatooine. I would understand if Kylo went there to live like Obi Wan, but Tatooine doesn't mean anything for Ray... They talked about it like the Skywalker loved it. I'm sure both Anakyn and Luke hated that place so much (for obvious reasons).
```

Comment:
R E Y

Reply:
It was to honor them (mostly Luke) for what I understood?

Reply:
> It was to honor them (mostly Luke) for what I understood?

In person reply xD